easybuild/easyconfigs/i/icc/icc-2017.1.132-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,33 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
+name = 'icc'
+version = '2017.1.132'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_%(version_major)s_update%(version_minor)s_composer_edition_for_cpp.tgz']
+
+checksums = ['41a0e8850ebb5f7169076c89be743ee2']
+
+gccver = '5.4.0'
+binutilsver = '2.26'
+versionsuffix = '-GCC-%s-%s' % (gccver, binutilsver)
+
+dependencies = [
+    ('GCCcore', gccver),
+    ('binutils', binutilsver, '', ('GCCcore', gccver)),
+]
+
+# list of regex for components to install
+# full list of components can be obtained from pset/mediaconfig.xml in unpacked sources
+# cfr. https://software.intel.com/en-us/articles/intel-composer-xe-2015-silent-installation-guide
+components = ['intel-comp', 'intel-ccomp', 'intel-icc', 'intel-openmp', 'intel-ipsc?_']
+
+dontcreateinstalldir = 'True'
+
+license_file = '/opt/easybuild/licenses/intel/license.lic'
```

Comment:
@vanzod this shouldn't be hardcoded like this, please use (same in other easyconfigs)

```python
license_file = HOME + '/licenses/intel/license.lic'
```